drivers/tee/optee/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+config OPTEE
+	bool "OP-TEE driver"
+	depends on TEE && (ARM64 || ZTEST)
```

Comment:
I see that `drivers/tee` dir is included to build only in case if we have enabled `CONFIG_TEE`, pls, look inside `drivers/CMakeLists.txt`
moreover you include this Kconfig only in case if TEE enabled, look inside `drivers/tee/Kconfig`

so, do we really need `TEE` dependency here?

Reply:
Removed.